commons/commons-http/src/main/kotlin/ebi/ac/uk/commons/http/slack/NotificationsSender.kt

```diff
@@ -6,7 +6,7 @@ import org.springframework.http.MediaType.APPLICATION_JSON
 import org.springframework.web.client.RestTemplate
 import org.springframework.web.client.postForEntity
 
-class NotificationsSender(
+open class NotificationsSender(
```

Comment:
I think rather than introduce extra classes and inhirantece we can use Spring @Qualifer to tag each bean diferently.